index.d.ts

```diff
@@ -41,10 +41,10 @@ export interface DropdownAlertProps {
   sensitivity?: number;
   defaultContainer?: object | number;
   defaultTextContainer?: object | number;
-  renderImage?(props: DropdownAlertProps, state: {type: string}): ReactNode;
-  renderCancel?(props: DropdownAlertProps, state: {type: string}): ReactNode;
-  renderTitle?(props: DropdownAlertProps, state: {type: string}): ReactNode;
-  renderMessage?(props: DropdownAlertProps, state: {type: string}): ReactNode;
+  renderImage?(props: DropdownAlertProps, state: RenderState): ReactNode;
```

Comment:
Technically, could define and reuse `AlertDataType` here since `this.alertData` is passed.

Reply:
`state: AlertDataType`

Reply:
Aha, makes sense to me -- updated